src/main/java/io/leonis/subra/ipc/serialization/protobuf/SSLCommandDeducer.java

```diff
@@ -0,0 +1,37 @@
+package io.leonis.subra.ipc.serialization.protobuf;
+
+import io.leonis.subra.game.data.Strategy.Supplier;
+import io.leonis.subra.protocol.Robot.Command;
+import io.leonis.zosma.game.engine.Deducer;
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Flux;
+
+/**
+ * The Class SSLCommandDeducer.
+ *
+ * Deduces a {@link io.leonis.subra.game.data.Strategy} to individual {@link Command Commands}.
+ *
+ * @author Jeroen de Jong
+ */
+public class SSLCommandDeducer implements Deducer<Supplier, Command> {
```

Comment:
might want to make that `Strategy.Supplier` explicitly just so it's a bit more understandable